.changes/new-change

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env python3
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+from argparse import ArgumentParser
+
+from tool import ChangeType
+from tool.new import new_change
+
+
+def main() -> None:
```

Comment:
Just a suggestion - after trying this tool out a bit locally it does seem really simple to use, but a small readme in this .changes directory showing how to use this in a common workflow would be helpful when down the road the commit message in this PR isn't as readily available

Reply:
Ah yeah I should add this to the contributing file